components/AddressInfos/AddressInfos.tsx

```diff
@@ -133,6 +134,7 @@ const AddressInfos: React.FC<Props> = ({ address }) => {
 
   const isUPRecovery = recoveryAddresses.includes(address);
   const isLSP1Delegate = Object.keys(LSP1_DELEGATE_VERSIONS).includes(address);
+  const isLSP1GraveForwarder = address == LSP1_GRAVE_FORWARDER;
```

Comment:
maybe you should put this in a useEffect + state or you might have issues when it updates the address